WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/SupportForm/SupportForm.swift

```diff
@@ -105,49 +106,110 @@ struct SupportForm: View {
     @StateObject var viewModel: SupportFormViewModel
 
     var body: some View {
-        VStack(spacing: Layout.sectionSpacing) {
-
-            HStack(spacing: -Layout.optionsSpacing) {
-                Text(Localization.iNeedHelp)
-                    .bold()
-                Picker(Localization.iNeedHelp, selection: $viewModel.area) {
-                    ForEach(viewModel.areas, id: \.self) { area in
-                        Text(area.title).tag(area)
+        ZStack() {
+            // Background
+            Color(.listBackground)
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+
+            VStack(spacing: .zero) {
+
+                // Scrollable Form
+                ScrollView {
+                    VStack(alignment: .leading, spacing: Layout.sectionSpacing) {
+
+                        Text(Localization.iNeedHelp.uppercased())
+                            .footnoteStyle()
+                            .padding([.horizontal, .top])
+
+                        // Area List
+                        VStack(alignment: .leading, spacing: .zero) {
+                            ForEach(viewModel.areas.indexed(), id: \.0.self) { index, area in
+                                HStack(alignment: .center, spacing: Layout.radioButtonSpacing) {
+                                    // Radio-Button emulation
+                                    Circle()
+                                        .stroke(Color(.separator), lineWidth: Layout.radioButtonBorderWidth)
+                                        .frame(width: Layout.radioButtonSize, height: Layout.radioButtonSize)
+                                        .background(
+                                            // Use a clear color for non-selected radio buttons.
+                                            Circle()
+                                                .fill( viewModel.isAreaSelected(area) ? Color(.accent) : .clear)
+                                                .padding(Layout.radioButtonBorderWidth)
+                                        )
+
+                                    Text(area.title)
+                                        .headlineStyle()
+                                }
+                                .padding()
+                                .frame(maxWidth: .infinity, alignment: .leading) // Needed to make tap area the whole width
+                                .background(Color(.listForeground(modal: false)))
+                                .onTapGesture {
+                                    viewModel.selectArea(area)
+                                }
+
+                                Divider()
+                                    .renderedIf(index < viewModel.areas.count - 1) // Don't render the last divider
```

Comment:
Should we also add padding to the leading edge of these dividers? It looks like that's in the design (so the leading edge of the divider matches the leading edge of the radio buttons).

Reply:
Good eye, fixed it on https://github.com/woocommerce/woocommerce-ios/pull/8869/commits/c7fee2fe4f7011e5c6b7d1daead58ecd2fb48460